pkgs/applications/networking/sync/rsync/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
   buildInputs = [libiconv zlib popt] ++ stdenv.lib.optional enableACLs acl;
```

Comment:
I think we should also remove zlib from `buildInputs` just in case and to avoid confusion.